.travis.yml

```diff
@@ -1,18 +1,12 @@
 language: python
 
-python: 3.5
+python: 3.7
 env:
-    - TOXENV=py27
-    - TOXENV=pypy
-    - TOXENV=py33
-    - TOXENV=py34
-    - TOXENV=py35
+    - TOXENV=py37
+    - TOXENV=py38
     - TOXENV=pep8
```

Comment:
Please add back support for Python 3.6.

Also, have a look at https://github.com/morepath/morepath_cerebral_todomvc/commit/c5b413728e05e92423c162534c32e6b39ee1cb0f where we fixed a similar problem, when CI failed as one Python interpreter was not installed. Thank you!

Reply:
Done